### PR TITLE
docs(main readme): added logging docs to main readme and out of speci…

### PR DESCRIPTION
**Pull request by swdpcomputing**
## Description of change

added logging docs to main readme and out of specific packages readmes

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [x] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -259,11 +259,28 @@ Kubernetes cluster.
...
+STDOut so they can be picked up when deployed by the Kubernetes pod(s). It
+is important to use the log levels (trace, debug, info, warn, error, fatal)
+appropriately so that filtering can be done during production.
+
+Also installed is `express-pino-logger` which add `req.log` to the incoming
+HTTP request. This automatically logs incoming HTTP requests and apply a unique
+correlation ID to make tracing errors through easier. For the most part, this
+should be the instance used for logging.
+
+Good practice for logging statements:
```
Comment (MrSimonEmms) [inline README.md:286]: Is this @todo?

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (swdpcomputing) [reply README.md]: Yes. Edited.

File: README.md [hunk 2]
```diff
@@ -259,11 +259,28 @@ Kubernetes cluster.
...
+STDOut so they can be picked up when deployed by the Kubernetes pod(s). It
+is important to use the log levels (trace, debug, info, warn, error, fatal)
+appropriately so that filtering can be done during production.
+
+Also installed is `express-pino-logger` which add `req.log` to the incoming
+HTTP request. This automatically logs incoming HTTP requests and apply a unique
+correlation ID to make tracing errors through easier. For the most part, this
+should be the instance used for logging.
+
+Good practice for logging statements: @todo
```
Comment (MrSimonEmms) [inline README.md:286]: Rather than putting this as a `@todo` in the code, can we actually get this done as part of this PR?

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]
If you need help doing the examples, put 30 mins in my diary and we can pair on it

File: README.md (hunk 2 above)
Comment (swdpcomputing) [reply README.md]: Done.

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]
Can you format the blobs of code by wrapping the lines with 3 backticks and with correct indentation? This should also help highlight any errors in the code. Eg

```
try {
  log.debug(<debug_object>, <debug_text>);
} catch (err) {
  log.error({ err }, 'error message');
  throw err;
}

```

**Review by MrSimonEmms — APPROVED** [review]

**Review by MrSimonEmms — APPROVED** [review]

**Pull request merged**